app/Http/Controllers/CarController.php

```diff
@@ -31,19 +31,20 @@ public function create()
     public function store(Request $request)
     {
         $data =$request->validate([
-          'carTitle' =>'required|string',
-          'description' =>'required|string|max:1000',
-          'price'=>'required|decimal:1',
+            'carTitle' =>'required|string',
+            'description' =>'required|string|max:1000',
+            'price'=>'required|decimal:1',
+            'image' => 'required|image|mimes:jpeg,png,jpg,gif'
         ]);
+        $imageName = time().'.'.$request->image->extension();
+        $request->image->move(public_path('images'), $imageName);
```

Comment:
you don't have to say go to public path as it by default see it `public_path('images')` ==> no need